### PR TITLE
refactor(frontend): remove migrate v1 in frontend plugin

### PR DESCRIPTION
**Pull request by XiaofuHuang**
Remove migrate v1 in frontend plugin

### CODE REVIEW

**Comment by XiaofuHuang** [comment]
E2E TEST: None

File: packages/fx-core/src/plugins/resource/frontend/plugin.ts [hunk 1]
```diff
@@ -235,32 +222,6 @@ export class FrontendPluginImpl implements PluginImpl {
     );
   }
 
-  public async executeUserTask(func: Func, ctx: PluginContext): Promise<TeamsFxResult> {
```
Comment (swatDong) [inline packages/fx-core/src/plugins/resource/frontend/plugin.ts:238]: Just realized - is `executeUserTask` part of plugin interface? Totally removing this method or just let it do nothing?

Comment (hund030) [reply packages/fx-core/src/plugins/resource/frontend/plugin.ts:238]: Both ok for me. If we do not need this interface now, just remove it.

Comment (XiaofuHuang) [reply packages/fx-core/src/plugins/resource/frontend/plugin.ts:238]: I addded `executeUserTask ` in PR: https://github.com/OfficeDev/TeamsFx/pull/2023
Before that frontend plugin does not contain the `executeUserTask ` lifecycle. @hund030 What do you think about it?

**Review by hund030 — APPROVED** [review]

**Pull request merged**